sdk/sdkclients/sdkruntimesclient/run.go

```diff
@@ -25,3 +31,57 @@ func (r *run) Call(context.Context, sdktypes.Value, []sdktypes.Value, map[string
 	// Need a way to pass this to the server - will do when the stream will be bidi.
 	return sdktypes.InvalidValue, sdkerrors.ErrNotImplemented
 }
+
+func (c *client) run(
+	ctx context.Context,
+	rid sdktypes.RunID,
+	path string,
+	build *sdkbuildfile.BuildFile,
+	globals map[string]sdktypes.Value,
+	cbs *sdkservices.RunCallbacks,
+) (sdkservices.Run, error) {
+	var a bytes.Buffer
+	if err := build.Write(&a); err != nil {
```

Comment:
General comment: If `BuildFile` was implementing `io.Reader` you can use `io.Copy`